daml-foundations/daml-ghc/tests/PartyVsText.daml

```diff
@@ -1,12 +1,11 @@
 -- @SINCE-LF 1.2
 daml 1.2
-module PartyFromText where
+module PartyVsText where
 
 import DA.Assert
 
 test = scenario do
   p <- getParty "alice"
-  show p === "'alice'"
```

Comment:
Why delete this test? It's still valid.

Reply:
I know it's duplicated, but I prefer seeing all party/text tests in one place.

Reply:
Makes sense. I've added it back.